Client/AuthRequestsTesterViewController.swift

```diff
@@ -12,6 +12,7 @@ import KarhooSDK
 enum AuthRequestType: String {
     case login = "Login"
     case revoke = "Revoke"
+    case adyen = "Adyen"
```

Comment:
perhaps better to call it adyenPaymentsMethods as the adyen itself is not very self-explanatory

Reply:
done